tests/bluetooth/tester/src/btp_gap.c

```diff
@@ -560,6 +561,28 @@ int tester_gap_stop_ext_adv(void)
 
 	return 0;
 }
+
+static uint8_t set_extended_advertising(const void *cmd, uint16_t cmd_len, void *rsp,
+					uint16_t *rsp_len)
+{
+	const struct btp_gap_set_extended_advertising_cmd *cp = cmd;
+	struct btp_gap_set_extended_advertising_rp *rp = rsp;
+
+	LOG_DBG("ext adv settings: %u", cp->settings);
+
+	if (cp->settings != 0) {
+		atomic_set_bit(&current_settings,
+			       BTP_GAP_SETTINGS_EXTENDED_ADVERTISING);
+	} else {
+		atomic_clear_bit(&current_settings,
+				 BTP_GAP_SETTINGS_EXTENDED_ADVERTISING);
+	}
```

Comment:
I know this is just copying existing code and thereby potentially out of scope to this PR, however such code patterns can generally be replaced by a single line:
```c
atomic_set_bit_to(&current_settings, BTP_GAP_SETTINGS_EXTENDED_ADVERTISING,
		  cp->settings != 0);
```